common/djangoapps/student/urls.py

```diff
@@ -40,6 +40,7 @@
     url(r'^course_run/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
         views.course_run_refund_status,
         name="course_run_refund_status"),
+    url(r'accounts/verify_password', views.verify_user_password, name='verify_password'),
```

Comment:
Nit: Can we group this with other password-related endpoints?  For example, move it above the `refund_status` endpoint?